### ISSUE TITLE
Integrate with minikube.

### ISSUE THREAD

**Issue by Random-Liu** [containerd/cri#462]
We should integrate with minikube to make it easier for people to try this out.

**Comment by AkihiroSuda** [comment]
@yanxuean @Random-Liu 
Any update? 😃

**Comment by Random-Liu** [comment]
@AkihiroSuda We punt this for now, because after CRI support is merged into containerd, this will be much simpler.

**Comment by Random-Liu** [comment]
Punt this to next release.

This will be much easier after we have a containerd release which has CRI built in.

**Comment by Random-Liu** [comment]
@yanxuean Let's do in containerd 1.2. I'll cherrypick #851 to v1.11.

**Comment by yanxuean** [comment]
ok

**Comment by yanxuean** [comment]
I have implement minikube for supporting containerd.  It base on containerd v1.1.2. The minikube need a containerd release version which must merge #851 of cri.  
I don't know what the due day of containerd 1.2 is.   Can we merge #851 to containerd v1.1.3?

**Comment by Random-Liu** [comment]
@yanxuean I'm fine with that to unblock the minikube integration effort.

/cc @containerd/containerd-maintainers

**Comment by crosbymichael** [comment]
@dmcgowan @Random-Liu do we have any other changes that would be in a 1.1.3?

**Comment by Random-Liu** [comment]
@crosbymichael That is the only one. :)

**Comment by yanxuean** [comment]
The pr for supporting containerd have merged. We will can use containerd in minikube v.0.28.3.
https://github.com/kubernetes/minikube/pull/3040
It is time to close this issue.

**Comment by Random-Liu** [comment]
@yanxuean https://github.com/containerd/containerd/releases/tag/v1.1.3 is available. You may want to update the minikube side correspondingly.

Sorry for the delay, keep distracted by something else. :P

**Comment by Random-Liu** [comment]
@yanxuean Ha, I see you are using 1.2.0-beta. That also works. :)

**Comment by yanxuean** [comment]
Yeah, The 1.2.0-beta is better.

**Comment by yanxuean** [comment]
a little pity, It is only based on a beta version. 
I will update the version after the official v1.2.0 is released.

**Comment by Random-Liu** [comment]
@yanxuean I don't have a machine with VirtualBox. Have you tried the containerd support in minikube yourself? It should work, right?

**Comment by yanxuean** [comment]
Yes,  I installed virtualbox on my ubuntu16.04  and try it.
